### PR TITLE
Update default version for TICK Stack.

### PR DESCRIPTION
**Pull request by gianarb**
All charts related to InfluxDB starts from an old version.

I just updated all of them.

### CODE REVIEW

**Comment by gianarb** [comment]
Hello!

I think this PR is ready to be reviewed. 

/assign @mattfarina

File: stable/chronograf/values.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## Image Settings
 ##
 image:
-  repository: "quay.io/influxdb/chronograf"
-  tag: "latest"
+  repository: "docker.io/chronograf"
+  tag: "1.3-alpine"
```
Comment (mattfarina) [inline stable/chronograf/values.yaml:5]: Thanks for moving this from latest to a tag.

File: stable/kapacitor/values.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ##
 image:
   repository: "kapacitor"
-  tag: "1.2"
+  tag: "1.4-alpine"
```
Comment (mattfarina) [inline stable/kapacitor/values.yaml:6]: Why the switch to alpine?

**Review by mattfarina — CHANGES_REQUESTED** [review]
Thank you for coming through and cleaning up the influxdb versions. But, according to the [contributing guidelines](https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md):

> NOTE: In order to make testing and merging of PRs easier, please submit changes to multiple charts in separate PRs.

Could you please separate this into one PR per chart? Fell free to assign them to me for review.

File: stable/kapacitor/values.yaml (hunk 2 above)
Comment (gianarb) [reply stable/kapacitor/values.yaml:6]: I added alpine to stay in line with the other images part of the stack. In charts, we are always using alpine. It is just for consistency.

**Comment by gianarb** [comment]
I close this PR in favour of this 3 as required by @mattfarina 
https://github.com/kubernetes/charts/pull/3186
https://github.com/kubernetes/charts/pull/3187
https://github.com/kubernetes/charts/pull/3188
https://github.com/kubernetes/charts/pull/3189

Thanks

**Pull request closed without merging**